setup.py

```diff
@@ -81,13 +82,15 @@ def get_short_description():
     return 'FSA/FST algorithms, intended to (eventually) be interoperable with PyTorch and similar'
 
 
+with open('k2/python/k2/__init__.py', 'a') as f:
+    f.write(f"__dev_version__ = '{get_package_version()}'\n")
```

Comment:
I am adding a new attribute to `k2` with this approach.
Perhaps @pzelasko you can also add an attribute to lhotse to contain its git sha1.

Reply:
Thanks I’ll check later if I can follow your approach

Reply:
I don't think this can work for Lhotse. It is easier in K2 because there is a C++ build that embeds the version + date every time you run it, so the Python code can query the native library for that. 

E.g. if I add a utility that queries git for sha1, it is going to crash when you `pip install lhotse`, because it is not a repository then. The best I can offer currently is `lhotse.__version__`, but it is going to change rarely. I'm not sure how other python packages deal with that, I'll look into it.

Reply:
I think it feasible.

I just created a demo repo to verify it. The repo is at the address https://github.com/csukuangfj/kk_test

If you install it using
```bash
pip install -vv git+https://github.com/csukuangfj/kk_test.git
```

After the installation, you will find a file `/some/path/to/your/venv/lib/python3.8/site-packages/kk_test/__init__.py`
containing
```
__version__ = '1.0.0'
__git_sha1__ = '96d9701521d62bf1bd127452e5aac6bc6029df79
```

---

Even though I don't show how to install it from PyPI, I think it is.always.possible to add
some file to `lhotse-x.x.x-py3-none-any.whl ` while you're building it or
add some file to `lhotse-x.x.x.tar.gz` when you create it.

The `some file` can contain whatever you want to put.  In the setup.py, it can figure out whether it
is able to get the `git sha1` from the repo; if not, just read it from `some file`.